### ISSUE TITLE
[FEAT] Snake Eating the Contribution Graph

### ISSUE THREAD

**Issue by Akshu-on-github** [Akshu-on-github/Make-your-own-README.md#6]
## Describe Addition
Add an example of the popular snake eating the GitHub Contribution Graph feature

### Additional context

<img src = "https://user-images.githubusercontent.com/61582763/135670495-10c7571e-19ee-4eb8-9582-97fc96b1ec6e.png" width = 50% align = "right">

**Step by step tutorial**: [ Watch a snake-eating🐍 my contribution graph on Github](https://dev.to/rado_mayank/watch-a-snake-eating-my-contribution-graph-on-github-96) by Mayank Choudhary

- [ ] I want this issue assigned to me
- [x] I have read the [Code of Conduct](CODE_OF_CONDUCT.md) and the [Contributing Guidelines](CONTRIBUTING.md)

**Comment by saksham-s1ngh** [comment]
I would like to work on this issue, if it's alright?

**Comment by Akshu-on-github** [comment]
Please go ahead @saksham-s1ngh!

**Comment by Akshu-on-github** [comment]
Hey there @saksham-s1ngh, any updates on this issue?

**Comment by saksham-s1ngh** [comment]
> Hey there @saksham-s1ngh, any updates on this issue?

Yes sorry, I was suddenly bogged by exams, will probably submit a pr by today or tomorrow.

**Comment by Akshu-on-github** [comment]
Oh, that's okay - just an interest check
